### DISCUSSION TITLE
making filesystes datapacks.

### DISCUSSION THREAD

**Discussion by blackdragon2447** [fnuecke/oc2#91] [Q&A] [answered]
is there an example/step by step on making filesystem datapacks?

**Comment by blackdragon2447** [comment, marked as answer]
update on the situation, i can get it to mount the filesystem provided by the addon, i do get an io error when accesing it tho.

**Reply by fnuecke** [reply]
Odd, feel free to open an issue then, I'll try to repro when I can. When you say addon, that means it's a datapack provided by another mod, instead of user-added, i.e. the living in the filesystem?

**Reply by blackdragon2447** [reply]
that is a mistake on my part, i meant a. resource pack yea.
